src/ResourceManager/Compute/Commands.Compute/Az.Compute.psd1

```diff
@@ -55,7 +55,7 @@ RequiredModules = @(@{ModuleName = 'Az.Profile'; ModuleVersion = '0.1.0'; })
 
 # Assemblies that must be loaded prior to importing this module
 RequiredAssemblies = '.\AutoMapper.dll', 
-    '.\Microsoft.Azure.Commands.Common.Strategies.4.dll', 
+    '.\Microsoft.Azure.PowerShell.Strategies.dll', 
```

Comment:
will need to rename once the assembly names are changed

Reply:
This will not change because this is the NetStandard psd1 for Compute.